docs/Coding-Guidelines/ErrorHandlingAndLogging.md

```diff
@@ -0,0 +1,17 @@
+# Error Handling and Logging
```

Comment:
like this - thx

Reply:
Made it simple - document the rules, then Uhoh! We have a violator! Fix that! :P